backend/controllers/backLink.ts

```diff
@@ -0,0 +1,37 @@
+import config from '../config'
+
+const sanitizeUrl = (url: string) => (url?.endsWith('/') ? url.substring(0, url.length - 1) : url)
+
+export interface RegisteredService {
+  name: string
+  hostname: string
+  defaultBackLinkText: string
+}
+
+export const registeredBackLinkServices = [
+  {
+    name: 'digital-prison-services',
+    hostname: '',
+    defaultBackLinkText: 'View most recent search',
+  },
+  {
+    name: 'welcome-people-into-prison',
+    hostname: sanitizeUrl(config.apis.welcomePeopleIntoPrison.url),
+    defaultBackLinkText: 'Back to Welcome people into prison',
+  },
+] as Array<RegisteredService>
+
+export const saveBackLink = (registeredServices: Array<RegisteredService>) => async (req, res) => {
```

Comment:
you could default this to the production `registeredBackLinkServices`, save exporting it to just inject it back in later and its a reasonable default.